src/ContosoCrafts.Web.Shared/Models/CartItem.cs

```diff
@@ -2,8 +2,8 @@ namespace ContosoCrafts.Web.Shared.Models
 {
     public class CartItem
     {
-        public int Price { get; set; }
         public int Quantity { get; set; }
         public string Title { get; set; }
+        public string Id { get; set; }
```

Comment:
I added the ID here so that the POST body included the ID of the product. That way, we can lookup the price on the server and use that as the price for the line item.